packages/plugin-ext/src/main/node/handlers/plugin-theia-directory-handler.ts

```diff
@@ -42,15 +42,20 @@ export class PluginTheiaDirectoryHandler implements PluginDeployerDirectoryHandl
 
     async accept(resolvedPlugin: PluginDeployerEntry): Promise<boolean> {
 
-        console.debug('PluginTheiaDirectoryHandler: accepting plugin with path', resolvedPlugin.path());
+        console.debug(`${this.constructor.name}: accepting plugin with path ${resolvedPlugin.path()}`);
```

Comment:
I believe `${this.constructor.name}` might not work after a production build, but it's a debug message anyway, so this might not be too important

Reply:
Ah, interesting point. I think probably it's trying too hard anyways. I'll just go with the generic name. Thanks!